### ISSUE TITLE
python sr.py  error

### ISSUE THREAD

**Issue by xuanzhangyang** [jiny2001/dcscn-super-resolution#21]
I downloaded the latest code，and ran 
```
python sr.py --file tmp.bmp
```
then i got some errors bellow

```
DCSCN v2-------------------------------------
2018/4/23 10:37:53 [dcscn_L12_F196to48_NIN_A64_PS_R1F32]
Feature:196 166 148 133 120 108 97 86 76 66 57 48  Total: (1301)64 32 32 384 1  Complexity:1,754,942 Receptive Fields:29
CNN1/conv_W:0 (3, 3, 1, 196)=1764, CNN1/conv_B:0 196, CNN1/prelu/CNN1_prelu:0 196, CNN2/conv_W:0 (3, 3, 196, 166)=292824, CNN2/conv_B:0 166, CNN2/prelu/CNN2_prelu:0 166, CNN3/conv_W:0 (3, 3, 166, 148)=221112, CNN3/conv_B:0 148, CNN3/prelu/CNN3_prelu:0 148, CNN4/conv_W:0 (3, 3, 148, 133)=177156, CNN4/conv_B:0 133, CNN4/prelu/CNN4_prelu:0 133, CNN5/conv_W:0 (3, 3, 133, 120)=143640, CNN5/conv_B:0 120, CNN5/prelu/CNN5_prelu:0 120, CNN6/conv_W:0 (3, 3, 120, 108)=116640, CNN6/conv_B:0 108, CNN6/prelu/CNN6_prelu:0 108, CNN7/conv_W:0 (3, 3, 108, 97)=94284, CNN7/conv_B:0 97, CNN7/prelu/CNN7_prelu:0 97, CNN8/conv_W:0 (3, 3, 97, 86)=75078, CNN8/conv_B:0 86, CNN8/prelu/CNN8_prelu:0 86, CNN9/conv_W:0 (3, 3, 86, 76)=58824, CNN9/conv_B:0 76, CNN9/prelu/CNN9_prelu:0 76, CNN10/conv_W:0 (3, 3, 76, 66)=45144, CNN10/conv_B:0 66, CNN10/prelu/CNN10_prelu:0 66, CNN11/conv_W:0 (3, 3, 66, 57)=33858, CNN11/conv_B:0 57, CNN11/prelu/CNN11_prelu:0 57, CNN12/conv_W:0 (3, 3, 57, 48)=24624, CNN12/conv_B:0 48, CNN12/prelu/CNN12_prelu:0 48, A1/conv_W:0 (1, 1, 1301, 64)=83264, A1/conv_B:0 64, A1/prelu/A1_prelu:0 64, B1/conv_W:0 (1, 1, 1301, 32)=41632, B1/conv_B:0 32, B1/prelu/B1_prelu:0 32, B2/conv_W:0 (3, 3, 32, 32)=9216, B2/conv_B:0 32, B2/prelu/B2_prelu:0 32, Up-PS/Up-PS_CNN/conv_W:0 (3, 3, 96, 384)=331776, Up-PS/Up-PS_CNN/conv_B:0 384, R-CNN1/conv_W:0 (3, 3, 96, 1)=864,
Total 48 variables, 1,754,942 params
Model initialized.
2018-04-23 10:38:07.455673: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key Up-PS/Up-PS_CNN/conv_B/Adam not found in checkpoint
2018-04-23 10:38:07.456275: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key Up-PS/Up-PS_CNN/conv_B/Adam not found in checkpoint
         [[Node: save/RestoreV2_139 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/cpu:0"](_arg_save/Const_0_0, save/RestoreV2_139/tensor_names, save/RestoreV2_139/shape_and_slices)]]
2018-04-23 10:38:07.456140: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key Up-PS/Up-PS_CNN/conv_B/Adam not found in checkpoint
         [[Node: save/RestoreV2_139 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/cpu:0"](_arg_save/Const_0_0, save/RestoreV2_139/tensor_names, save/RestoreV2_139/shape_and_slices)]]
2018-04-23 10:38:07.457115: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key Up-PS/Up-PS_CNN/conv_B/Adam not found in checkpoint
         [[Node: save/RestoreV2_139 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/cpu:0"](_arg_save/Const_0_0, save/RestoreV2_139/tensor_names, save/RestoreV2_139/shape_and_slices)]]
2018-04-23 10:38:07.457231: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key Up-PS/Up-PS_CNN/conv_B/Adam not found in checkpoint
         [[Node: save/RestoreV2_139 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/cpu:0"](_arg_save/Const_0_0, save/RestoreV2_139/tensor_names, save/RestoreV2_139/shape_and_slices)]]
2018-04-23 10:38:07.457489: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key Up-PS/Up-PS_CNN/conv_B/Adam not found in checkpoint
         [[Node: save/RestoreV2_139 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/cpu:0"](_arg_save/Const_0_0, save/RestoreV2_139/tensor_names, save/RestoreV2_139/shape_and_slices)]]
2018-04-23 10:38:07.457527: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key Up-PS/Up-PS_CNN/conv_B/Adam not found in checkpoint
         [[Node: save/RestoreV2_139 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/cpu:0"](_arg_save/Const_0_0, save/RestoreV2_139/tensor_names, save/RestoreV2_139/shape_and_slices)]]
2018-04-23 10:38:07.457530: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key Up-PS/Up-PS_CNN/conv_B/Adam not found in checkpoint
         [[Node: save/RestoreV2_139 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/cpu:0"](_arg_save/Const_0_0, save/RestoreV2_139/tensor_names, save/RestoreV2_139/shape_and_slices)]]
2018-04-23 10:38:07.457596: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key Up-PS/Up-PS_CNN/conv_B/Adam not found in checkpoint
         [[Node: save/RestoreV2_139 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/cpu:0"](_arg_save/Const_0_0, save/RestoreV2_139/tensor_names, save/RestoreV2_139/shape_and_slices)]]
2018-04-23 10:38:07.457714: W tensorflow/core/framework/op_kernel.cc:1192] Not found: Key Up-PS/Up-PS_CNN/conv_B/Adam not found in checkpoint
         [[Node: save/RestoreV2_139 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/cpu:0"](_arg_save/Const_0_0, save/RestoreV2_139/tensor_names, save/RestoreV2_139/shape_and_slices)]]

```

**Comment by msiraj83** [comment]
I aslo face the same problem as shown in screen shot.
![tensorflow_error](https://user-images.githubusercontent.com/8863047/39111964-34f3758a-4701-11e8-88bc-c1954f094a10.png)

**Comment by jiny2001** [comment]
Hi, I believe some variable names might be corrupted.

I updated models for x2 and x3. Please try it. Will upload x4 soon later.
Thx.

**Comment by xuanzhangyang** [comment]
The problem is solved，thinks. Would you like to share some trick? Use bsd200+yang91 dataset, I can get results similar to yours, but when i retrain with other dataset, I get a bad result. Learning rate, dataset preprocess and so on, do you have some trick？

**Comment by jiny2001** [comment]
There is no trick. I tried other datasets and just found that the datasets has a better feature for set5/set14/bsd100. 

Also DIV2K with more training iterations will have a better performance, I believe.
